src/cascadia/TerminalApp/SuggestionsControl.cpp

```diff
@@ -851,11 +851,18 @@ namespace winrt::TerminalApp::implementation
                 }
             }
         }
-        if (_mode == SuggestionsMode::Palette)
-        {
-            // We want to present the commands sorted
-            std::sort(actions.begin(), actions.end(), FilteredCommand::Compare);
-        }
+
+        // TODO (for discussion):
+        //
+        // Do we want sorting in the palette? I have it disabled, so results are
+        // filtered, but still in the original order. This seems more correct,
+        // but I could be convinced otherwise. It could maybe be a setting.
+
+        // if (_mode == SuggestionsMode::Palette)
+        // {
+        //     // We want to present the commands sorted
+        //     std::sort(actions.begin(), actions.end(), FilteredCommand::Compare);
+        // }
```

Comment:
BARE TODO

I vote keep original order and we see how this feels in a bug bash or after some initial feedback.

Reply:
I know we weren't able to bash this in the most recent one, but in the 1.18 bashes that included this, original order felt good IMO. Cause then it's basically in history order

Reply:
Cool, then yeah let's go with original order!